core/domain/skill_services_test.py

```diff
@@ -1059,7 +1114,7 @@ def test_migrate_skill_contents_to_latest_schema(self):
         model.commit(
             'user_id_admin', 'skill model created', commit_cmd_dicts)
 
-        swap_skill_object = self.swap(skill_domain, 'Skill', MockSkillObject)
+        swap_skill_object = self.swap(skill_domain, 'Skill', skill_domain.Skill)
```

Comment:
Is this an expected change?

Reply:
Yes we have removed the `MockSkillObject`

Reply:
Then remove th line completely? [i.e, dont mock!]

Reply:
done